### PR TITLE
Feature/new analysis endpoint

### PR DESCRIPTION
**Pull request by pjosh**
## Overview

This PR changes the analysis url for the `umd-loss-gain` dataset to use the need endpoint that @01painadam made here: https://github.com/gfw-api/gfw-umd-forest-api/pull/8
I've also added a new select to analyze admin2 regions.

## Demo

admin2

### CODE REVIEW

File: app/assets/javascripts/services/CountryService.js [hunk 1]
```diff
@@ -42,142 +50,217 @@ define([
...
-          },
-          error: function(errors) {
-            reject(errors);
-          }
-        };
-
-        this.abortRequest(GET_REQUEST_COUNTRIES_LIST_ID);
-        this.currentRequest[GET_REQUEST_COUNTRIES_LIST_ID] = ds.request(requestConfig);
-      }.bind(this));
+      return new Promise(
```
Comment (edbrett) [inline app/assets/javascripts/services/CountryService.js]: Can we make sure these are alphabetically sorted?

Comment (edbrett) [reply app/assets/javascripts/services/CountryService.js]: Currently Norway is first.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "globalforestwatch",
-  "version": "3.0.1",
+  "version": "3.1.0",
```
Comment (edbrett) [inline package.json]: Not sure why this is here?

Comment (pjosh) [reply package.json]: I don't know either 🤷‍♂️

Comment (simaob) [reply package.json]: 1 and 0 swapped places because 1 was tired of being on the window seat?

**Comment by edbrett** [comment]
Additional comments:
- When there is no analysis data (Check norway for example) we should give feedback
- We are still missing the 2010 data in the analysis. Maybe you could discuss with JC how to handle this?

**Pull request merged**